hack/verify-multiarch-test-images.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"
+NON_MULTIARCH_IMAGES=()
+
+for IMAGE_DIR in $(find test/images -iname 'Dockerfile' -printf '%h\n'| LC_ALL=C sort -u)
+do
+    grep -q "^${IMAGE_DIR}$" hack/.multiarch_images_exceptions.txt && ret=$? || ret=$?
+    if [[ "${ret}" -ne 0 ]]; then
+        if [[ ! -f "${IMAGE_DIR}/VERSION" ]]; then
```

Comment:
I'd like to enforce more rules here, but I think you should split this up into a separate func and just pass the image dir to validate.
e.g.
 - VERSION exists and is a `1.x` version
 - Dockerfile exists
 - something else?